services/gundeck/test/unit/ThreadBudget.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE NoGeneralizedNewtypeDeriving #-}
```

Comment:
why?  and if this is part of ghc2010, should we disable it globally instead?

Reply:
I'm surprised they released GHC2021 with that extension on. Removing it everywhere might be wise.